packages/opencensus-exporter-stackdriver/src/stackdriver-monitoring.ts

```diff
@@ -61,14 +68,16 @@ export class StackdriverStatsExporter implements StatsEventListener {
    * Is called whenever a measure is recorded.
    * @param views The views associated with the measure
    * @param measurement The measurement recorded
+   * @param metricPrefix The metric prefix
```

Comment:
When would the user want to provide a prefix here or in `onRegisterView`?

Reply:
The user can provide as argument for class and optional in onRegisterView
https://github.com/census-instrumentation/opencensus-node/pull/151/commits/d1eaedb18730ffd4a1545a612eff529b35692b21#diff-2bc66aa8590ce821951700582ff8e483